interfaces/Glitter/templates/static/javascripts/glitter.main.js

```diff
@@ -691,6 +691,9 @@ function ViewModel() {
         var file = files[fileindex]
         fileindex++
 
+        // Check if it's maybe a folder, we can't handle those
+        if(!file.type && file.size % 4096 == 0) return;
```

Comment:
According to the docs, files that are actually folders are reported without a type and a size that's exact multiples of `4096`. No idea why, but it works :P